ava.base.js

```diff
@@ -1,9 +1,14 @@
+import { exec } from 'child_process'
 import fs, { existsSync } from 'fs'
 import path from 'path'
 import process from 'process'
 
 import { isCI } from 'ci-info'
 
+if (process.argv.includes('-w')) {
+  exec('tsc -w')
```

Comment:
have you tested if the process get's cleaned up properly? I would use `execa` for it

Reply:
NIT: I would use `npm run build:dev` or something like that (so that we have to maintain that build command just in one place) and please check the process cwd (I assume it's correct)